awx/api/metadata.py

```diff
@@ -238,12 +238,10 @@ def get_field_info(self, field):
         res = super(JobTypeMetadata, self).get_field_info(field)
 
         if field.field_name == 'job_type':
-            index = 0
-            for choice in res['choices']:
-                if choice[0] == 'scan':
-                    res['choices'].pop(index)
-                    break
```

Comment:
you lost functionality by removing the `break`, but if this was only going to have like 2 elements and you say you did it on purpose, then I will believe you.

Reply:
Not sure I follow.  Isn't the point of the *original* code to loop over the list and remove the first item that's "scan" (and then bail out)?

My version builds a new list that filters *out* scan, so it's not modifying a data structure while it's being iterated over.

Reply:
yep, and I see the bigger picture now. It wasn't a profound comment.

before, it could have (hypothetically) transformed `["run", "scan", "scan"]` to `["run", "scan"]`, which is an implementation detail, because the list should have never had the same thing twice, and your new code is 1000 times better in every way.